docs/builds/guides/frameworks/react.md

```diff
@@ -481,14 +519,17 @@ Then add two new elements to the exported object under the `module.rules` array
 				}
 			}
 		},
+		'css-loader',
```

Comment:
Why did you change this? CRA1 comes with an old postcss-version so its very unlikely that anyone using it will be also using the latest version of postcss-loader.

Reply:
If anything, I'd link here to the CRA2+ section saying:

```
<info-box>
	If you use `postcss-loader` in version 4+ see LINK
</info-box>
```